app/src/main/java/org/mozilla/fenix/compose/button/RadioButton.kt

```diff
@@ -0,0 +1,107 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose.button
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.RadioButtonColors
+import androidx.compose.material.RadioButtonDefaults
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Radio Button.
+ *
+ * @param selected [Boolean] showing if the radio button is selected or not.
+ * @param modifier [Modifier] to be applied to the radio button.
+ * @param enabled [Boolean] that controls if radio button is selectable.
+ * @param onClick Callback to be invoked when radio button is clicked.
+ */
+@Composable
+fun RadioButton(
+    selected: Boolean,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    colors: RadioButtonColors = RadioButtonDefaults.colors(
+        selectedColor = FirefoxTheme.colors.iconActionSecondary,
+        unselectedColor = FirefoxTheme.colors.iconActionSecondary,
+        disabledColor = FirefoxTheme.colors.iconDisabled,
```

Comment:
I don't think these color tokens are right. In https://www.figma.com/file/ujdiReypSypu2GhRgzArea/Android-Components?node-id=3022%3A18104&t=fGDALFypru3dkcyW-0, it looks like we are using the form default and form selected.

Reply:
ah good catch thank you! fixed.